### DISCUSSION TITLE
[Bug]: Not working with Laravel Sail

### DISCUSSION THREAD

**Discussion by chrispmcgee** [spatie/laravel-pdf#37] [Ideas]
### What happened?

I received an error when using the package in its most basic form on a project using Laravel Sail.

### The command:
$pdf = \Spatie\LaravelPdf\Facades\Pdf::view('pdfs.dispatch-sheet', (new DispatchSheetResource($this->load))->resolve())
            ->format(Format::Letter)
            ->save('Dispatch_Sheet_PO_' . $this->load->po_number . '.pdf');

### The error:
![firefox_unfiCriZSl](https://github.com/spatie/laravel-pdf/assets/137904126/751168d8-f009-4924-ab0a-887ef8b983e2)

### How to reproduce the bug

I am pretty sure this occurred because I am using Laravel Sail, which has its own PHP environment in a Docker container that cannot launch the headless Chromium browser process needed to convert the blade template to a PDF.

The steps to reproduce:

1. Create or open a Laravel project using Sail for the local development environment.
2. Install and use Laravel-PDF 1.1.0 to create a PDF document.

### Package Version

1.1.0

### PHP Version

8.2.14

### Laravel Version

10.39.0

### Which operating systems does with happen with?

Linux

### Notes

I intend to use Laravel-PDF to generate a PDF attached to a simple mail message using a Laravel Notification inside a synchronous Job. It's not getting that far yet, as the `->save()` method throws the above error before it gets to the Notification part of the Job.

**Comment by freekmurze** [comment]
I don't see this as a bug in our code. If you figure out how to run this package within Sail, feel free to PR the instructions to our docs.

**Comment by elijahstorm** [comment]
Commenting just to say I am also having the same issue and unable to configure it to run with Sail.

**Comment by alfonsogarza** [comment]
For those wondering: 

You have to configure chromium in your docker and let browsershot know about it

I successfully followed these instructions:
https://laracasts.com/discuss/channels/laravel/run-browsershot-in-laravel-sail-on-arm64-mac

Via Pdf you can access the Browsershot instance

```
          ->withBrowsershot(function (Browsershot $browsershot) {
                $browsershot
                    ->setNodeBinary('/usr/bin/node')
                    ->setNpmBinary('/usr/bin/npm')
                    ->setRemoteInstance('172.23.0.2');
            });
```


PS. Does anyone know why all issues are transformed into discussions? Just curious.
